metadata-ingestion/src/datahub/ingestion/source/gc/datahub_gc.py

```diff
@@ -140,36 +161,61 @@ def create(cls, config_dict, ctx):
     def get_workunit_processors(self) -> List[Optional[MetadataWorkUnitProcessor]]:
         return [partial(auto_workunit_reporter, self.get_report())]
```

Comment:
Maybe it is a bit better if we just create our own `auto_workunit_reporter` and use that instead of the one which report warning